Palace/Reader2/UI/TPPEPUBViewController.swift

```diff
@@ -197,3 +208,22 @@ extension TPPEPUBViewController: UIPopoverPresentationControllerDelegate {
   }
 }
 
+extension TPPEPUBViewController: EPUBSearchDelegate {
+  func didSelect(location: Locator) {
+    
+    defer {
+      presentedViewController?.dismiss(animated: true)
+      navigator.go(to: location)
+    }
+  
+    if let navigator = navigator as? DecorableNavigator {
+      
+      var decorations: [Decoration] = []
+      decorations.append(Decoration(
+        id: "tts-search",
+        locator: location,
+        style: .highlight(tint: .red)))
+      navigator.apply(decorations: decorations, in: "tts")
```

Comment:
may be we should use `search` here, as `tts` means text-to-speach

Reply:
ah I see, thanks, will update